lists/is/is-ken.json

```
@@ -0,0 +1,61 @@
+{
+  "name": {
+    "en": "Icelandic Social Security Number",
+    "local": "Íslensk kennitala"
+  },
+  "url": "https://www.skra.is/",
+  "description": {
+    "en": "**Identification number - kennitala**. \nIn Iceland, persons and enterprises are issued a unique identification number which is recorded in the national register and register of enterprises.\n\nIndividuals are identified with the ID numbers in the national registry system. There are two types of personal ID number registered in the national registry i.e. system ID No. and personal ID No.  \nPersonal ID numbers are issued at birth to all children born in Iceland and Icelandic citizens born abroad. Personal ID numbers are also assigned to all individuals who register their domicile in Iceland.\nA system ID number is only issued to individuals who intend to stay less than 3-6 months in Iceland or do not intend to stay in the country at all. \n"
+  },
+  "coverage": [
+    "IS"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "company"
+  ],
+  "sector": [],
+  "code": "",
```

```suggestion
  "code": "IS-KEN",
```
Validation is failing as shown here: https://deploy-preview-526--org-id-register.netlify.app/type/lists/record/is-ken/ because the list needs a code value.